client/src/components/Languages.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { LangContainer, TableTitle } from './styles/styles';
+
+const Languages = (props) => {
+  console.log('props in languages', props)
+  let check = '&#10004;'
+  return (
+    <div>
+      <LangContainer>
+        <TableTitle>Languages:</TableTitle>
+        <table>
+          <tbody>
+            <tr>
+              <th></th>
+              <th>Interface</th>
+              <th>Full Audio</th>
+              <th>Subtitles</th>
+            </tr>
+            <tr>
+              <td>{props.languages[0]}</td>
```

Comment:
Nice use of conditional rendering. Other than the indexes it looks like everything else is being reused. Is there any way to avoid this or is it just the nature of the logic?